README.md

```diff
@@ -1,250 +1,326 @@
+
+All branches whose name complies to the "X.Y.Z" pattern are built and put into
+the "stable" downloads directory:
 [http://buildbot.berlin.freifunk.net/buildbot/stable/](http://buildbot.berlin.freifunk.net/buildbot/stable/)
 
-All branches with names not fitting the "X.Y.Z" pattern are built and put into the "unstable" directory:
+All branches with names not fitting the "X.Y.Z" pattern are built and put into
+the "unstable" directory:
 [http://buildbot.berlin.freifunk.net/buildbot/unstable/](http://buildbot.berlin.freifunk.net/buildbot/unstable/)
-Note that in the directory there is no reference to the branch name; unstable builds can be identified by build number only.
+Note that in the directory there is no reference to the branch name; unstable
+builds can be identified by their build number only.
+
+For development images you will find, that the hash of the commit they are based
+on is included in their file name.
+
 
 #### Creating a release
 
-Every release has a [semantic version number](http://semver.org); each major version has its own codename.
-We name our releases after important female computer scientists, hackers, etc.
-For inspiration please take a look at the related
-[ticket](https://github.com/freifunk-berlin/firmware/issues/24).
+Every release has a [semantic version number](http://semver.org); each major
+version has its own codename. We name our releases after important female
+computer scientists, hackers, etc. For inspiration please take a look at the
+related [ticket](https://github.com/freifunk-berlin/firmware/issues/24).
 
-For a new release, create a new branch. The branch name must be a semantic version
-number. Make sure you change the semantic version number and, for major releases,
-the codename in the README and config files (./configs/*)
+For a new release, create a new branch. The branch name must be a semantic
+version number. Make sure you change the semantic version number and, for major
+releases, the codename in the README and config file (`configs/common.config`).
 
-The buildbot will build the release and place the files in the stable direcotry
+The buildbot will build the release and place the files in the stable directory
 once you pushed the new branch to github.
 
+
 ### Patches with "git format-patch"
 
 **Important:** all patches should be pushed upstream!
 
-If a patch is not yet included upstream, it can be placed in the corresponding subdirectory below the`patches`
-directory. To create a correct patch-file just use the [`git format-patch`](https://git-scm.com/docs/git-format-patch) command.
+If a patch is not yet included upstream, it can be placed in the corresponding
+subdirectory below the `patches` directory. To create a correct patch-file just
+use the [`git format-patch`](https://git-scm.com/docs/git-format-patch) command.
+
 
 #### Create a patch
 
 In order to add a patch file update your build environment by running:
 
-```bash
```

Comment:
on all the following code-blocks you broke the layout. please compare with the current file.

Reply:
reversed.